.github/renovate.json5

```diff
@@ -1,93 +1,81 @@
 {
-  $schema: 'https://docs.renovatebot.com/renovate-schema.json',
   extends: [
-    ':combinePatchMinorReleases',
-    ':separateMajorReleases',
-    ':semanticCommits',
-    ':semanticPrefixChore',
-    ':semanticCommitScopeDisabled',
-    ':updateNotScheduled',
-    ':automergeRequireAllStatusChecks',
-    ':ignoreUnstable',
-    ':gitSignOff',
     'group:monorepos',
     'helpers:disableTypesNodeMajor'
   ],
+
+  labels: ['dependencies'],
+  rangeStrategy: 'bump',
+  prHourlyLimit: 2,
+  prConcurrentLimit: 20,
+  updateNotScheduled: true,
+  commitBody: 'Signed-off-by: {{{gitAuthor}}}',
   ignorePaths: [
     '**/node_modules/**',
     '**/__tests__/**',
     '**/test/**'
   ],
-  force: {
-    constraints: {
-      npm: '>=7'
-    },
-  },
```

Comment:
I remembered that we needed this to force NPM v7 instead of v6 so that v2 `package-lock.json` would be generated. Has this been solved by Renovate?

Reply:
Renovate already detects this value in the package-json.